tests/room_test.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+from ..main.app import Amity
+import sqlite3
+
+import os
+
+
+
+class roomstest(unittest.TestCase):
+    def setUp(self):
+        self.amity = Amity()
+        self.conn = sqlite3.connect("test_amity.sqlite")
+        self.connect = self.conn.cursor()
+        self.connect.execute(
```

Comment:
**Testing: Nitpick**

You should not be running SQL logic of this sort directly within your tests. You should call a function that does this.